app/templates/admin/all_agencies.html

```diff
@@ -0,0 +1,89 @@
+                    <!--</div>-->
+                    <!--<i class="dropdown icon"></i>-->
+                    <!--<div class="menu">-->
+                        <!--<div class="item" data-value="">All account types</div>-->
+                        <!--{% for r in roles %}-->
+                            <!--<div class="item" data-value="{{ r.name }}">{{ r.name }}s</div>-->
+                        <!--{% endfor %}-->
+                    <!--</div>-->
+                <!--</div>-->
+                <div class="ui right search item">
+                    <div class="ui transparent icon input">
+                        <input id="search-users" type="text" placeholder="Search agencies…">
+                        <i class="search icon"></i>
+                    </div>
+                </div>
+            </div>
+
+            {# Use overflow-x: scroll so that mobile views don't freak out
+             # when the table is too wide #}
+            <div style="overflow-x: scroll;">
+                <table class="ui searchable sortable unstackable selectable celled table">
+                    <thead>
+                        <tr>
+                            <th class="sorted ascending">Name</th>
+                            <th>Approved by Admin</th>
+                            <th>Publicly Visible</th>
+                            <th>Number Of Reports</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                    {% for a in agencies | sort(attribute='name') %}
+                        <tr onclick="window.location.href = '{{ url_for('admin.agency_info', agency_id=a.id) }}';">
+
+                            <td>{{ a.name }}</td>
+                            <td>{{ 'Yes' if a.is_official else 'No' }}</td>
+                            <td>{{ 'Yes' if a.is_public else 'No' }}</td>
+                            <td>{{ a.reports|length }}</td>
+                        </tr>
+                    {% endfor %}
+                    </tbody>
+                </table>
+            </div>
+        </div>
+    </div>
+
+    <script type="text/javascript">
+        $(document).ready(function () {
+            $('#search-users').keyup(function () {
+                var searchText = $(this).val();
+                if (searchText.length > 0) {
+                    $('tbody td:icontains(' + searchText + ')').addClass('positive');
+                    $('td.positive').not(':icontains(' + searchText + ')').removeClass('positive');
+                    $('tbody td').not(':icontains(' + searchText + ')').closest('tr').addClass('hidden').hide();
+                    $('tr.hidden:icontains(' + searchText + ')').removeClass('hidden').show();
+                } else {
+                    $('td.positive').removeClass('positive');
+                    $('tr.hidden').removeClass('hidden').show();
+                }
+            });
+
+            $('#select-role').dropdown({
```

Comment:
hm i guess we don't have anything populating in the dropdown yet right? if we are taking out the dropdown, i'd also get rid of this fxn. otherwise maybe add a TODO instead